### ISSUE TITLE
this repo need a dedicated library

### ISSUE THREAD

**Issue by apatlpo** [NoeLahaye/ITideNATL#1]
agreed?

If yes, I propose to initiate one in branch averaging.

**Comment by apatlpo** [comment]
I proceeded further ... it can always be adjusted later on

also we should soon merge branch averaging in master

**Comment by NoeLahaye** [comment]
Agreed. I just don't have a clear idea of the scope of the library right now. Some of the processing will heavily rely on xorca (starting from computing the stratification, vertical mode and projecting), some of it does not (averaging)

**Comment by NoeLahaye** [comment]
> also we should soon merge branch averaging in master

I think we can do it ASAP, since averaging scripts are operational

**Comment by apatlpo** [comment]
scope will be limited but useful ... in order to avoid code duplication
you'll be able to see in a second

**Comment by NoeLahaye** [comment]
library initiated with #2
